src/content/resources/architectural-overview.md

```diff
@@ -1023,58 +1047,71 @@ supports, there are some unique characteristics of Flutter's web support that
 are worthy of comment.
 
 Dart has been compiling to JavaScript for as long as the language has existed,
-with a toolchain optimized for both development and production purposes. Many
-important apps compile from Dart to JavaScript and run in production today,
+with a toolchain optimized for both development and production purposes.
+Many important apps compile from Dart to JavaScript and run in production today,
 including the [advertiser tooling for Google Ads](https://ads.google.com/home/).
-Because the Flutter framework is written in Dart, compiling it to JavaScript was
-relatively straightforward.
+Because the Flutter framework is written in Dart, compiling it to JavaScript
+was relatively straightforward.
 
 However, the Flutter engine, written in C++,
 is designed to interface with the
 underlying operating system rather than a web browser.
 A different approach is therefore required.
-On the web, Flutter provides a reimplementation of the
-engine on top of standard browser APIs.
-We currently have two options for
-rendering Flutter content on the web: HTML and WebGL.
-In HTML mode, Flutter uses HTML, CSS, Canvas, and SVG.
-To render to WebGL, Flutter uses a version of Skia
-compiled to WebAssembly called
-[CanvasKit](https://skia.org/docs/user/modules/canvaskit/).
-While HTML mode offers the best code size characteristics,
-`CanvasKit` provides the fastest path to the
-browser's graphics stack,
-and offers somewhat higher graphical fidelity with the
-native mobile targets[^4].
-
-The web version of the architectural layer diagram is as follows:
 
-![Flutter web
-architecture](/assets/images/docs/arch-overview/web-arch.png){:width="100%"}
+On the web, Flutter offers two build mode and two renderers.
```

Comment:
modes

Reply:
Weird, this line was changed many commits ago.

Reply:
I take it back. I was accidentally doing these last grammatical fixes on the main branch!!!!